bioblend/_tests/TestGalaxyUsers.py

```diff
@@ -61,3 +61,27 @@ def test_get_current_user(self):
         self.assertIsNotNone(user['email'])
         self.assertIsNotNone(user['nice_total_disk_usage'])
         self.assertIsNotNone(user['total_disk_usage'])
+
+    def test_update_user(self):
+        # WARNING: only admins can create users!
+        # WARNING: Users cannot be purged through the Galaxy API, so execute
+        # this test only on a disposable Galaxy instance!
+        if self.gi.config.get_config()['use_remote_user']:
+            self.skipTest('This Galaxy instance is not configured to use local users')
+        new_user_email = 'newuser2@example.com'
+        user = self.gi.users.create_local_user('newuser2', new_user_email, 'secret')
+        self.assertEqual(user['username'], 'newuser2')
+        self.assertEqual(user['email'], new_user_email)
+
+        updated_user_email = 'updateduser@example.com'
+        updated_username = 'updateduser'
+        current_user = user['id']
+        user = self.gi.users.update_user(current_user, username=updated_username, email=updated_user_email)
+        user = self.gi.users.show_user(current_user)
+        self.assertEqual(user['username'], updated_username)
+        self.assertEqual(user['email'], updated_user_email)
+
+        if self.gi.config.get_config()['allow_user_deletion']:
+            deleted_user = self.gi.users.delete_user(user['id'])
+            self.assertEqual(deleted_user['email'], new_user_email)
+            self.assertTrue(deleted_user['deleted'])
```

Comment:
The 2 asserts are probably not needed.

Reply:
Any way the first assert is wrong, the 2nd parameter should be `updated_user_email`